tests/Application/config/packages/_sylius.yaml

```
@@ -13,3 +13,6 @@ parameters:
 sylius_shop:
     product_grid:
         include_all_descendants: true
+
+sylius_api:
+    enabled: true
```

```suggestion
    enabled: false
```
In this context i think we could disable it, we don't need it